bounce.py

```
@@ -26,20 +26,26 @@
-        logging.info(f"Valid domain extracted: {domain}")
-        domain_counter_stat.labels(domain).inc()
-        redirect_url = bounce_redirect_url + domain
-    else:
-        logging.warning(f"Invalid domain: {domain}, redirecting to root.")
-        domain_counter_stat.labels("root").inc()
-
-    return redirect(redirect_url, code=301)
+    try:
+        # Extract the domain from the Host header
+        domain = request.headers.get('Host', 'unknown')
+        logging.debug(f"Received request for domain: {domain}")
```

Ruff (0.7.0)

Logging statement uses f-string

(G004)